From: Bruno Haible <bruno@clisp.org>
Subject: Bump po-mode version
Origin: upstream, https://git.savannah.gnu.org/gitweb/?p=gettext.git;a=commitdiff;h=303a8f98c4e73025addf8ee629db00182585bd43
Bug-Debian: https://bugs.debian.org/1055020

    * gettext-tools/emacs/po-mode.el (po-mode-version-string): Bump.

--- a/gettext-tools/emacs/po-mode.el
+++ b/gettext-tools/emacs/po-mode.el
@@ -62,7 +62,7 @@
 
 ;;; Code:
 
-(defconst po-mode-version-string "2.27" "\
+(defconst po-mode-version-string "2.28" "\
 Version number of this version of po-mode.el.")
 
 ;;; Emacs portability matters - part I.
